lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/test.ts

```diff
@@ -0,0 +1,107 @@
+}
+
+// The compiler throws an error if the function is provided a first argument which is not an array of nested arrays...
+{
+	const shapes: [ Array<number>, Array<number>, Array<number> ] = [ [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ] ];
+	bternary5d( 'abc', shapes, fcn ); // $ExpectError
+	bternary5d( 3.14, shapes, fcn ); // $ExpectError
+	bternary5d( true, shapes, fcn ); // $ExpectError
+	bternary5d( false, shapes, fcn ); // $ExpectError
+	bternary5d( null, shapes, fcn ); // $ExpectError
+	bternary5d( [ '1' ], shapes, fcn ); // $ExpectError
+	bternary5d( {}, shapes, fcn ); // $ExpectError
+	bternary5d( ( x: number ): number => x, shapes, fcn ); // $ExpectError
+}
+
+// The compiler throws an error if the function is provided a second argument which is not an array of arrays...
+{
+	const x = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const y = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const z = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const out = [ [ [ [ [ 0.0, 0.0 ], [ 0.0, 0.0 ] ] ] ] ];
+
+	bternary5d( [x, y, z, out ], 'abc', fcn ); // $ExpectError
+	bternary5d( [x, y, z, out ], 3.14, fcn ); // $ExpectError
+	bternary5d( [x, y, z, out ], true, fcn ); // $ExpectError
+	bternary5d( [x, y, z, out ], false, fcn ); // $ExpectError
+	bternary5d( [x, y, z, out ], null, fcn ); // $ExpectError
+	bternary5d( [ x, y, z, out ], [ '1' ], fcn ); // $ExpectError
+	bternary5d( [ x, y, z, out ], {}, fcn ); // $ExpectError
+	bternary5d( [ x, y, z, out ], ( x: number ): number => x, fcn ); // $ExpectError
+}
+
+// The compiler throws an error if the function is provided a third argument which is not a valid callback...
+{
+	const x = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const y = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const z = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const out = [ [ [ [ [ 0.0, 0.0 ], [ 0.0, 0.0 ] ] ] ] ];
+
+	const shapes: [ Array<number>, Array<number>, Array<number> ] = [ [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ] ];
+	bternary5d( [ x, y, z, out ], shapes, 'abc' ); // $ExpectError
+	bternary5d( [ x, y, z, out ], shapes, 3.14 ); // $ExpectError
+	bternary5d( [ x, y, z, out ], shapes, true ); // $ExpectError
+	bternary5d( [ x, y, z, out ], shapes, false ); // $ExpectError
+	bternary5d( [ x, y, z, out ], shapes, null ); // $ExpectError
+	bternary5d( [ x, y, z, out ], shapes, [ '1' ] ); // $ExpectError
+	bternary5d( [ x, y, z, out ], shapes, {} ); // $ExpectError
+}
+
+// The compiler throws an error if the function is provided an unsupported number of arguments...
+{
+	const x = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const y = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const z = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const out = [ [ [ [ [ 0.0, 0.0 ], [ 0.0, 0.0 ] ] ] ] ];
+
+	const shapes: [ Array<number>, Array<number>, Array<number> ] = [ [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ] ];
+	bternary5d(); // $ExpectError
```

Comment:
```suggestion

	bternary5d(); // $ExpectError
```

Consistency in the use of whitespace based on L43.